multic/cli/FeatureExtraction/FeatureExtraction.py

```diff
@@ -75,17 +76,19 @@ def main(args):
 
 
     all_comparts = [gloms,s_gloms,tubs, arts]
-    all_columns = [['x1','x2','y1','y2','Area','Mesangial Area','Mesangial Fraction'],
-                   ['x1','x2','y1','y2','Area','Mesangial Area','Mesangial Fraction'],
-                   ['x1','x2','y1','y2','Average TBM Thickness','Average Cell Thickness','Luminal Fraction'],
-                   ['x1','x2','y1','y2','Arterial Area']]
+    all_columns = [['x1','x2','y1','y2','Area (micron^2)','Mesangial Area (micron^2)','Mesangial Fraction'],
```

Comment:
@sayatmimar need to print `mpp` as without that `micron^2` is difficult to understand. Maybe a separate column for `mpp`. It's a constant though for a given slide, so I don't know how to display it without repetition, but the feature consumer needs to know the `mpp` to make sense of the pathomic features.

Reply:
@anindya-paul I converted everything to pixels and removed `mpp`. Do you think we should still add the value in the excel spreadsheet? Mpp usually appears on the image file in DSA see below, and there's an API to get this info as well.
<img width="730" alt="Screen Shot 2024-04-25 at 4 41 39 PM" src="https://github.com/SarderLab/Multi-Compartment-Segmentation/assets/131045129/a8e8f271-9bcd-4bc3-9c84-2caf67a5707f">

Reply:
Thanks! Yes, I saw that in `Large Image Metadata` before. Generally, our users are folks with medical background and most of the time, they'll overlook the `Large Image Metadata` table. However, as we're not using the mpp in our feature calculations, my suggestion doesn't matter anymore. Only the confusing part in `Large Image Metadata` is why it is displayed in `mm_x (milli-micron-x)` and `mm_y (milli-micron-y)` with so many decimals, much easier to understand in `mpp_x` and `mpp_y` which is `0.25` considering `magnification=40X`.